vdm-io/Joomla-Component-Builder
# Fix Database IF NOT EXISTS issue for older databases and updated marked.js (vulnerability)

Pull Request for Issue gh-841.

### Summary of Changes
- Updates marked JS package.
- Adds more events to the compiler.
- Fix gh-841 database issue (IF NOT EXISTS).
- Update Agerix banner link.
- Adds more clone options.

### Testing Instructions
Check that the database issue is resolved and that you can clone files_folders, component_modules, component_plugins, component_placeholders, component_dashboards. Also check that all notices that get converted with marked.js loads correctly.
